app/views/deploy_mailer/deployment_email.html.erb

```diff
@@ -0,0 +1,23 @@
+<body>
+  <p>
+    The following tickets have been deployed:
+  </p>
+  <div style="position:absolute; left:-9999px; top:0px;">
+    <table class="table table-striped table-deploy-email" id="jira_issues_clipboard">
+      <thead class="table-deploy-email-header">
+        <tr class="table-deploy-email-body">
+          <th class="table-deploy-email-header-row">Key</th>
+          <th class="table-deploy-email-header-row" style="min-width: 300px">Summary</th>
+        </tr>
+      </thead>
+      <tbody class="table-deploy-email-body">
+      <% @jira_issues.each_with_index do |issue, index| %>
+        <tr class="table-deploy-email-body <%= index % 2 == 0 ? 'table-deploy-email-body-shadow' : '' %>">
```

Comment:
I think you can use `cycle` here, so then could also get rid of the `with_index` and yielding "index"

```
<tr class="table-deploy-email-body <%= cycle('', 'table-deploy-email-body-shadow') %>">
```